test/cobalt/experiments/test_rerank_quality.py

```
@@ -0,0 +1,226 @@
+            reranked = _simulate_rerank(test_case['query'], test_case['documents'])
+
+            mock_rocketride_client.run_pipeline.return_value = {
+                'status': 'completed',
+                'output': {'reranked_documents': reranked},
+            }
+
+            # The top document should have high true relevance
+            top_doc = reranked[0]
+            assert top_doc['true_relevance'] >= 0.8, f'Top reranked document for "{test_case["query"]}" has low true relevance: {top_doc["true_relevance"]}'
+
+    def test_relevance_monotonically_decreasing(self, mock_rocketride_client):
```

Ruff (0.15.7)

[warning] Unused method argument: `mock_rocketride_client`

(ARG002)